github-actions/oss-readme-template/__tests__/main.test.ts

```diff
@@ -1,28 +1,20 @@
-import {wait} from '../src/wait';
-import * as process from 'process';
-import * as cp from 'child_process';
-import * as path from 'path';
-
-test('throws invalid number', async () => {
-  const input = parseInt('foo', 10);
-  await expect(wait(input)).rejects.toThrow('milliseconds not a number');
+test('placeholder', () => {
+  expect(true).toEqual(true);
 });
 
-test('wait 500 ms', async () => {
-  const start = new Date();
-  await wait(500);
-  const end = new Date();
-  const delta = Math.abs(end.getTime() - start.getTime());
-  expect(delta).toBeGreaterThan(450);
-});
+// This is an example test that came from the github actions template.  It shows
+// how to spawn a process that runs our `main` in the way that github will do it.
+// might be good to have some tests that cover this, but at present, our main
+// unconditionally writes a README.md file on the filesystem, which we don't want
+// to do in tests.
 
 // shows how the runner will run a javascript action with env / stdout protocol
-test('test runs', () => {
-  process.env['INPUT_MILLISECONDS'] = '500';
-  const np = process.execPath;
-  const ip = path.join(__dirname, '..', 'lib', 'main.js');
-  const options: cp.ExecFileSyncOptions = {
-    env: process.env,
-  };
-  console.log(cp.execFileSync(np, [ip], options).toString());
-});
+// test('test runs', () => {
+//   process.env['INPUT_MILLISECONDS'] = '500';
+//   const np = process.execPath;
+//   const ip = path.join(__dirname, '..', 'lib', 'main.js');
+//   const options: cp.ExecFileSyncOptions = {
+//     env: process.env,
+//   };
+//   console.log(cp.execFileSync(np, [ip], options).toString());
+// });
```

Comment:
mean to leave commented? Maybe just remove dummy test since you added your own?

Reply:
yeah I meant to leave that in, I would never be able to figure out how to write that test without an example if we ever do want to resuscitate it.  I put a comment above explaining why I was leaving it in.